src/status_im2/contexts/chat/home/view.cljs

```diff
@@ -54,12 +54,7 @@
       [:<>
```

Comment:
its not possible to comment unchanged code, so pointer is wrong, but we need to move contacts-section-header from here to contact-list